README.md

```diff
@@ -2,6 +2,67 @@
 
 This repository provides a GitHub Action for running the [Kani Rust Verifier](https://github.com/model-checking/kani) in CI.
 
+## GitHub Action Parameters
+
+The following parameters can be used to configure and customize the behavior of this GitHub Action:
+
+NOTE: All the fields provided are optional and have default behaviors when not specified.
+
+`kani-version`
+
+- **Description**: Specifies the Kani version number to use.
+- **Default**: 'latest'
+- **Example-pinned-version**: `'0.33.0'`
```

Comment:
What's the purpose of this item? I think it's better to provide examples for the most popular cases in the documentation.